simplified-books-borrowing/src/main/java/org/nypl/simplified/books/borrowing/BorrowAcquisitions.kt

```diff
@@ -20,7 +21,9 @@ object BorrowAcquisitions {
     entry: OPDSAcquisitionFeedEntry
   ): OPDSAcquisitionPath? {
     val paths = OPDSAcquisitionPaths.linearize(entry)
-    val filtered = paths.filter { support.isSupportedPath(it.asMIMETypes()) }
-    return filtered.firstOrNull() ?: return null
+    return paths
+      .filter { support.isSupportedPath(it.asMIMETypes()) }
+      // when borrowing a book, we need to ignore if the acquisition is for a sample/preview
+      .firstOrNull { it.source.relation != OPDSAcquisition.Relation.ACQUISITION_SAMPLE }
```

Comment:
Shouldn't this also exclude "preview"?

Reply:
I didn't add the "preview" because we still haven't moved on with the samples feature. I replied to your other comment, but basically the "preview" code was pushed by mistake.

Reply:
Ah, that makes sense.